daemon/daemon.go

```diff
@@ -455,20 +455,24 @@ func policyCommitMessage(us policy.Updates, cause update.Cause) string {
 func policyEventTypes(u policy.Update) []string {
 	types := map[string]struct{}{}
```

Comment:
I don't see the point of using an empty struct here. Why not just an array?

Just looked at the rest of the code. It eventually returns an array anyway, so just swap this out to be an array.